web/components/PetPostModal/PetPostModal.tsx

```diff
@@ -203,18 +207,37 @@ import {
   ageLabels,
   fosterTypeDescriptions,
 } from "../../utils/types/post";
+import { trpc } from "../../utils/trpc";
+import { Role } from "../../utils/types/account";
+import { useAuth } from "../../context/auth";
 
 const PetPostModal: React.FC<{
   isOpen: boolean;
   onClose: () => void;
-  postData: IPost;
+  postData: IPost & { _id: ObjectId };
 }> = ({ isOpen, onClose, postData }) => {
   const {
     isOpen: isFormViewOpen,
     onOpen: onFormViewOpen,
     onClose: onFormViewClose,
   } = useDisclosure();
 
+  const {
+    isOpen: isDeleteConfirmationOpen,
+    onOpen: onDeleteConfirmationOpen,
+    onClose: onDeleteConfirmationClose,
+  } = useDisclosure();
+
+  const { userData } = useAuth();
+  const role = userData?.role;
+
+  const posts = trpc.post.getAllPosts.useQuery().data;
+  console.log(posts);
```

Comment:
Assuming this was used for testing - could we remove this without affecting functionality?